docs/source/mechanical_analyzers/rotor_speed_limit_analyzer.rst

```diff
@@ -0,0 +1,163 @@
+.. _rotor_speed_analyzer:
+
+
+SPM Rotor Speed Analyzer
+##############################
+
+This analyzer determines the failure speed of a given surface-mounted permanent magnet (SPM) rotor design.  
+
+Model Background
+****************
+
+This analyzer utilizes the **SPM Rotor Structural Analyzer** to determine the rotor internal stresses at a given range of rotational speed. For determining failure point, maximum shear stress theory (MSST) and von Mises failure criterion are used. 
+To provide context, ductile material such as rotor shaft and rotor lamianation, von Mises failure criterion was used in evaluating failure. For brittle material, such as adhesive and permanent magnet, MSST failure criterion [#]_ was used.
+
+Detailed description of the **SPM Rotor Structural Analyzer** can be found on the analyzer page `here <https://emach.readthedocs.io/en/latest/mechanical_analyzers/SPM_structural_analyzer.html#inputs-from-user>`_.
+
+.. [#]  Ideally, for brittle materials, Mohr–Coulomb theory should be used in determining failure. However, due to the lack of material data avaliable to us, MSST is used instead.
+
+Inputs from User
+**********************************
+
+The SPM rotor speed problem class requires the dimensions of the rotor shaft, the rotor material dictionary ``mat_dict``, the maximum RPM ``N_max`` to evaluate problem up to , and material failure dictionary ``mat_failure_dict``.
+For the required rotor shaft dimensions and rotor material dictionary ``mat_dict``, please refer to the `Input from User  <https://emach.readthedocs.io/en/latest/mechanical_analyzers/SPM_structural_analyzer.html>`_ section in the **SPM Rotor Structural Analyzer** documentation page.
```

Comment:
`please refer to the Input from User section` Let's update this link so that when you click on it, it takes you directly to that section.